addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -619,6 +621,7 @@
             var self = this;
             var slide = this.get('slide');
             self._pushUrlState();
+            if (this.websiteAnimateWidget) this.websiteAnimateWidget = null;
```

Comment:
This writing style is discouraged and we usually use brackets and put this on 3 lines, eases readability. 

Would also move it inside "_renderSlide", just after we empty the "$content", as it's where we "re-initialize the container", so it would make more sense?

Last thing, aren't we supposed to specifically call "destroy" on such widgets? Can't remember if it's the case in 15.0, I let you double-check.
(So you'd call destroy and then set the reference to null).

Reply:
@awa-odoo Thanks again for the review :) Indeed it makes sense what you're proposing, this should be good to go If I didn't missed something.